odoo/tools/config.py

```diff
@@ -552,6 +552,8 @@ def die(cond, msg):
             self.options['pg_path'] = opt.pg_path
 
         self.options['test_enable'] = bool(self.options['test_tags'])
+        if self.options['test_tags']:
```

Comment:
I believe the correct way to determine whether "we are in test mode" is
```suggestion
        if self.options['test_enable'] or self.options['test_file']:
```